apps/chrome-extension/src/app/ui/model/useSetup.ts

```diff
@@ -1,7 +1,8 @@
+import browser from 'webextension-polyfill'
 import { textPreferencesStatePersist } from '@/entities/textPreferences'
 import { StorageLocal } from '@/shared/lib/storage'
 
 export function useSetup() {
-  const storage = chrome.extension ? chrome.storage.local : new StorageLocal()
+  const storage = browser.extension ? browser.storage.sync : new StorageLocal()
```

Comment:
I think we should stick with the `browser.storage.local` to save the preferences, since it allows storing big size data.

The problem with `browser.storage.sync` is that each item can [store up to 8KB of data](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync). For example having 2 profiles already takes 8KB.

<img width="501" alt="Screenshot 2023-01-20 at 17 06 09" src="https://user-images.githubusercontent.com/1078719/213746942-2d83c07f-786f-447f-92d3-9469503681af.png">

Reply:
Yes you're right. I thought it was an mistake because there are some others place where we use `sync` to access for `textAdaptationPreferences` key

Reply:
Initially the preferences were stored in `.local` because of the size, but other settings (which are mostly flags) are in `.sync`.